code/game/machinery/doors/door.dm

```diff
@@ -48,6 +48,9 @@
 	var/polarized_glass = FALSE
 	var/polarized_on
 
+	/// Are we barricaded? Stops the door from opening if so
+	var/door_barricaded = 0
```

Comment:
```suggestion
	var/door_barricaded = FALSE
```

Reply:
We can only have one barricade per door, right?

Reply:
yes, but admins can still spawn stuff, if making it a number affects performance in a horrible way or causes unrelated bugs i'll remove it but right now i don't see a reason to